app/src/main/java/com/github/se/orator/ui/friends/AddFriends.kt

```diff
@@ -184,60 +228,197 @@ fun AddFriendsScreen(
 }
 
 /**
- * Composable function that represents a single user item in a list. Displays the user's profile
- * picture, name, and bio, and allows adding the user as a friend.
+ * Composable function that represents a single sent friend request item in the list.
+ *
+ * It displays the friend's profile picture, name, bio, and a button to cancel the request.
+ *
+ * @param sentRequest The [UserProfile] object representing the user to whom the request was sent.
+ * @param userProfileViewModel The [UserProfileViewModel] that handles friend request cancellation.
+ */
+@Composable
+fun SentFriendRequestItem(sentRequest: UserProfile, userProfileViewModel: UserProfileViewModel) {
+  val context = LocalContext.current
+
+  Surface(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(horizontal = AppDimensions.smallPadding)
+              .clip(RoundedCornerShape(AppDimensions.roundedCornerRadius))
+              .testTag("sentFriendRequestItem#${sentRequest.uid}"),
+      color = AppColors.LightPurpleGrey,
+      shadowElevation = AppDimensions.elevationSmall // Subtle shadow with low elevation
+      ) {
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(AppDimensions.paddingMedium),
+            verticalAlignment = Alignment.CenterVertically) {
+              // Friend's Profile Picture
+              ProfilePicture(
+                  profilePictureUrl = sentRequest.profilePic,
+                  onClick = { /* Optionally, show enlarged picture */})
+              Spacer(modifier = Modifier.width(AppDimensions.smallWidth))
+              Column(
+                  modifier = Modifier.weight(1f), // Expand to push Cancel button to the end
+                  verticalArrangement = Arrangement.Center) {
+                    // Friend's Name
+                    Text(
+                        text = sentRequest.name,
+                        style = MaterialTheme.typography.titleMedium,
+                        modifier =
+                            Modifier.padding(bottom = AppDimensions.smallPadding)
+                                .testTag("sentFriendRequestName#${sentRequest.uid}"))
+                    // Friend's Bio
+                    Text(
+                        text = sentRequest.bio ?: "No bio available",
+                        style = MaterialTheme.typography.bodySmall,
+                        color = AppColors.secondaryTextColor,
```

Comment:
The colors used should come from `MaterialTheme.colorScheme` to be linked with the centralized color manager.
```suggestion
                        color = MaterialTheme.colorScheme.secondary,
```